src/main/java/cloud/eppo/EppoHttpClient.java

```diff
@@ -41,34 +42,23 @@ private static OkHttpClient buildOkHttpClient() {
     return builder.build();
   }
 
-  // TODO: use this for Java, callback for Android; clean as needed
-  public Response get(String path) {
-    HttpUrl httpUrl =
-        HttpUrl.parse(baseUrl + path)
-            .newBuilder()
-            .addQueryParameter("apiKey", apiKey)
-            .addQueryParameter("sdkName", sdkName)
-            .addQueryParameter("sdkVersion", sdkVersion)
-            .build();
+  public byte[] get(String path) {
```

Comment:
wrt Streaming the data, unfortunately, we do need to load the entire contents into memory in order to parse it due to the nature of serialized JSON.

Reply:
can you expand on "due to the nature of serialized JSON"? Just curious about where the issue is

Reply:
I stand corrected, thank you @felipecsl for pushing and teaching :) Really do appreciate it.
Now that we've established we can in fact use a byte stream, should we do it in this PR, a followup, or keep it on the backburner?

Reply:
i'm okay keeping in the backburner. not a blocker from my perspective. thanks for not dropping the thread :)